src/main/java/com/ivan/salesapp/services/ProductService.java

```diff
@@ -29,21 +33,54 @@ public ProductServiceModel addProduct(ProductServiceModel productServiceModel) {
 
     @Override
     public List<ProductServiceModel> findAllProducts() {
-        return null;
+        return this.productRepository.findAll().stream()
+                .map(p -> this.modelMapper
+                        .map(p, ProductServiceModel.class)).collect(Collectors.toList());
     }
```

Comment:
format this -> return this.productRepository.findAll() //
                                 .stream() //
                                 .map(p -> this.modelMapper.map(p, ProductServiceModel.class)) //
                                 .collect(toList());